operator/runner.go

```diff
@@ -49,17 +52,22 @@ func NewRunner(cfg RunnerConfig) (*Runner, error) {
 		cfg.Port = 9090
 	}
 
+	hc := cfg.HealthCheck
+	if isInterfaceNil(hc) {
```

Comment:
not sure if there is a util elsewhere I can use for this, but go is pretty bad at checking an interface value for nil.

Reply:
`nil` check will fail if one passes an empty interface value inadvertently - which will be a runtime memory exception when invoking methods on that interface.

To pass an empty interface value, all you got to do in Go is:

```go
var someVal HealthCheck
passThisToSomeFuncWhichOnlyChecksNil(someVal) // crashes
```

Reply:
Changed to a simple nil check for now.